module/documents/item.mjs

```diff
@@ -1527,6 +1527,9 @@ export default class Item5e extends SystemDocumentMixin(Item) {
     const elvenAccuracy = (flags.elvenAccuracy
       && CONFIG.DND5E.characterFlags.elvenAccuracy.abilities.includes(this.abilityMod)) || undefined;
 
+    // Targets
+    const targets = this.constructor._formatAttackTargets();
```

Comment:
I notice that `formatAttackTargets` still uses the actor name rather than the token name. It would be good to use the token name in cases where DMs are using auto-numbering (`Goblin #1` & `Goblin #2`, rather than just `Goblin` & `Goblin`).